java-symbol-solver-core/src/test/java/com/github/javaparser/symbolsolver/model/typesystem/TypeVariableUsageTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Federico Tomassetti
```

Comment:
As someone who does a lot of projects I would say: who cares, I only look at these things if I want to ask questions :-)